### PR TITLE
v10.0-prealpha

### PR DESCRIPTION
**Pull request by brysondev**

### CODE REVIEW

File: src/blog/2024-11-02/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+## New features
+
+### Deathmatch
+
+Deathmatch is the same as TDM with both CTG capture points & ghosts disabled, and enables respawning. 
+
+The new ConVars for these settings are as follows:
+
+* `neo_vote_game_mode`- 1 by default, Vote on game mode to play. TDM=0, CTG=1, VIP=2, **DM=3**
```
Comment (brysondev) [inline src/blog/2024-11-02/index.md]:
```
`neo_vote_game_mode` -
```
I've fixed this. Awaiting any other change requests before merging.

File: src/blog/2024-11-02/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+title: Pre-Alpha v10.0 Release Note
+summary: Release note for Neotokyo; Rebuild Pre-Alpha v9.0
```
Comment (nullsystem) [inline src/blog/2024-11-02/index.md]: "Rebuild Pre-Alpha v9.0" -> "Rebuild Pre-Alpha v10.0"

File: src/blog/2024-11-02/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+* [Download v10.0-prealpha build](https://github.com/NeotokyoRebuild/neo/releases/tag/v10.0-prealpha)
+* [Install NT;RE (client)](/guide/install/)
+* [GitHub Issues (Bug reports and feature requests)](https://github.com/NeotokyoRebuild/neo/issues)
+
+## New features
+
+### Deathmatch
+
+Deathmatch is the same as TDM with both CTG capture points & ghosts disabled, and enables respawning.
```
Comment (nullsystem) [inline src/blog/2024-11-02/index.md]: Make it more clear it's free for all, win by either XP limit (default 50) or time out (default 10 mins)

File: src/blog/2024-11-02/index.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+* `neo_sv_dmspawn_create` - Create a dmspawn spawn.
+* `neo_sv_dmspawn_removeall` - Remove all dmspawn spawns.
+* `neo_sv_dmspawn_printlocs` - Print all coordinates of the dmspawn spawns.
+* `neo_sv_dmspawn_save` - Save the map's loc dmspawn file.
+* `neo_sv_dmspawn_load` - Load the map's loc dmspawn file.
+* `neo_sv_dmspawn_teleportnext` - Teleport to the next dmspawn spawn.
+* `neo_sv_dmspawn_mapinfo` - Check amount of dmspawn spawns and if it can DM.
+
+If you wish to customize the spawns for a dedicated server, be sure to copy the generated dmspawn .loc files to the server.
```
Comment (nullsystem) [inline src/blog/2024-11-02/index.md]: Clarify that they will appear and read from under `/neo/maps/dm_locs`

File: src/blog/2024-11-02/index.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+* Added `cl_neo_bullet_trace_max_pen` (cheat protected) - How much pen does a bullet need to have to show up as a solid red line. Configure to the current weapon used, or leave on default to see differences in pen between weapons
+* Rangefinder restored from original NEOTOKYO
+    * Added `neo_cl_hud_rangefinder_enabled` - 1 as default, toggle enable/disable
+    * Added `neo_cl_hud_rangefinder_pos_frac_[x/y]` - 0.55 as default, position in fraction in screen width/height
+* Corrected Jinrai recon sprinting animations
+* Compass fade commands have been restored
+* Fixed SRS weapon reload timing
+* Added `neo_sv_gamemode_enforcement` - Default = 0 (map - neo_game_config), determine gamemode by... 0 = map (neo_game_config), 1 = neo_sv_gamemode_single, 2 = Random within neo_sv_gamemode_random_allow, 3 = vote (neo_vote_game_mode in idle/warmup)
+* Added `neo_sv_gamemode_single` - The single game mode to enforce
+* Added `neo_sv_gamemode_random_allow` - The bitwise of game modes that is allowed to be used for the randomizer
```
Comment (nullsystem) [inline src/blog/2024-11-02/index.md]: Put in: `* Gamemode enforcement by default is now changed to by map. Singular/random/by vote also implemented.` (something like this, not exact grammatically)

Then sub-section the convars bullet points

File: src/blog/2024-11-02/index.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+### Clan Tags
+
+Clan Tags can now be set in the options menu, or by typing `neo_clantag <tag>` in the developer console. Server owners can opt out of clan tags being show by adding `neo_sv_clantag_allow 0` to their server configuration files. If all players on a team have a matching clan tag, the team's name will reflect the tag set.
+
+### Viewmodel Only Lean
+
+Players now have the option to enable viewmodel only leaning. This prevents a roll effect to the camera view while leaning. Option can be changed in the options menu, or by typing `cl_neo_lean_viewmodel_only 1` in the developer console.
+
+## General fixes/changes
```
Comment (nullsystem) [inline src/blog/2024-11-02/index.md]: Can you split the fixes and new addition, this bullet point section is quite long.

**Review by nullsystem — CHANGES_REQUESTED** [review]

File: src/blog/2024-11-02/index.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+* [Download v10.0-prealpha build](https://github.com/NeotokyoRebuild/neo/releases/tag/v10.0-prealpha)
+* [Install NT;RE (client)](/guide/install/)
+* [GitHub Issues (Bug reports and feature requests)](https://github.com/NeotokyoRebuild/neo/issues)
+
+## New features
+
+### Deathmatch
+
+Deathmatch is the free-for-all equivalent to TDM with both CTG capture points & ghosts disabled, and enables respawning. Win by achieving 50 XP or after the timer runs out (10.25 minutes)
```
Comment (AdamTadeusz) [inline src/blog/2024-11-02/index.md]: ghost disabled

**Review by AdamTadeusz — APPROVED** [review]

**Review by nullsystem — APPROVED** [review]

**Pull request merged**